compiler/arm/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1448,6 +1448,18 @@ OMR::ARM::TreeEvaluator::mstoreiEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
    }
 
+TR::Register*
+OMR::ARM::TreeEvaluator::mToLongBitsEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   }
+
+TR::Register*
+OMR::ARM::TreeEvaluator::mLongBitsToMaskEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   }
+
```

Comment:
It seems the definitions of the following evaluators are missing for 32 bit ARM.
mTrueCountEvaluator, mFirstTrueEvaluator, mLastTrueEvaluator

Reply:
Have you updated your commit?

Reply:
The following evaluators are still missing for 32 bit ARM.
mTrueCountEvaluator, mFirstTrueEvaluator, mLastTrueEvaluator

Reply:
You are right! Somehow missed again. I think all should be there now. Sorry about that.